Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -506,6 +506,7 @@ def bottle_formula(formula)
               tab.write
             end
 
+            Homebrew.install_bundler_gems!(groups: ["bottle"])
```

Comment:
Can this go right to the top of `run`, next to the other existing call? It's nice to do this (so it succeeds or fails) before we do anything meaningful.